todo/src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux'
+import { createStore } from 'redux'
+import reducers from './reducers'
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+
+
+const store = createStore(reducers);
+
+store.subscribe(() => console.log(store.getState()))
+
+ReactDOM.render(
+    <Provider store={store}>
+        <App />
+    </Provider>
+    , document.getElementById('root'));
```

Comment:
It's preferred to keep indentation even, so for the opening parenthesis on line 13, we want one below on line 18. So:
```javascript
ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
);